x-pack/plugin/ml-package-loader/build.gradle

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
```

Comment:
X-Pack code is _not_ supposed to be dual licensed - it should be Elastic license v2 only. You can copy the appropriate header from the next file down in the PR.